.github/workflows/build_and_test.yml

```diff
@@ -5061,12 +5061,7 @@ jobs:
       id: cache-artifact
       uses: actions/cache@v2.1.4
       with:
-        path: |
-          winflexbison/bin/Debug
-          wsbuild
-          wireshark-win64-libs*
-          vcpkg-qt/installed
-          ${{ github.job }}.tar.xz
+        path: ${{ github.job }}.tar.xz
```

Comment:
Does this need an "extract-archive" step directly after this one?

In the first revision of this (before dealing with tar files manually), we were depending on the Cache Artifact action to put things back where they were.

Reply:
I don't think so. If the cache action is a hit, it should download the tar.xz file and skip every step except uploading it as an artifact. Extracting the wireshark artifact is done in the job 'build_w19-release_j_ws_FM-1f'